cli/compiler.rs

```diff
@@ -160,6 +160,18 @@ fn req(specifier: &str, referrer: &str, cmd_id: u32) -> Buf {
   .into_boxed_bytes()
 }
 
+pub fn get_compiler_config(
```

Comment:
Can you add a triple slash doc explaining the return values? It wasn't immediately clear to me what the two components of `(String, Vec<u8>)`  represented